environment.yml

```diff
@@ -5,3 +5,5 @@ channels:
 dependencies:
   - mkdocs=1.3.0
   - mkdocs-material=8.5.2
+  - matplotlib
```

Comment:
It probably makes sense to have these be part of a separate env file.

Reply:
What's the best way to do that and also keep the ability to create a Binder for this repo? I tried looking on the Binder reference but it isn't clear to me how/where I could specify a different `environment.yml` file... @taylorreiter would you happen to know how to do this?